src/main/java/edu/ucsb/cs156/example/controllers/CollegiateSubredditController.java

```diff
@@ -1,151 +0,0 @@
-package edu.ucsb.cs156.example.controllers;
```

Comment:
Deleting these files might create conflicts when the other guys work on the CollegiateSubreddit frontend.
Not sure why you are deleting them though.
If you decide to proceed with these deletions, let alvin and wenjin know beforehand or create an issue to reimplement these